src/cartservice/src/cartservice.csproj

```diff
@@ -7,16 +7,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.AspNetCore" Version="2.53.0" />
-    <PackageReference Include="Grpc.AspNetCore.HealthChecks" Version="2.53.0" />
-    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs" Version="1.5.0-rc.1" />
```

Comment:
This package was intentionally remove. It is currently part of OpenTelemetry.Exporter.OpenTelemetryProtocol (stable version)